syft/frameworks/torch/mpc/falcon/falcon_helper.py

```diff
@@ -67,6 +82,47 @@ def xor(
             result = result.wrap()
         return result
 
+    @staticmethod
+    def determine_sign(
+        x: ReplicatedSharingTensor, beta: ReplicatedSharingTensor
+    ) -> ReplicatedSharingTensor:
+        """
+        return: x if beta = 0 | -x if beta = 1
+        """
+        beta_ = 1 - beta * 2
+        # The order is important if the ring sizes are different.
+        # The left operand's ring size takes precedence.
+        return x * beta_
+
+    @staticmethod
+    def private_compare_preprocess(players, p=7, k=10):
```

Comment:
This method is never used, 
it can be useful and save space to have such a method (see the repeated preprocess code in tests )

Reply:
We do need the `private_compare_preprocess` to generate the `beta` and `m` mask